assets/src/wizards/subscriptions/index.js

```diff
@@ -40,31 +44,65 @@ class SubscriptionsWizard extends Component {
 	 * Render the example stub.
 	 */
 	render() {
-		const { inputTextValue1, inputTextValue2 } = this.state;
+		const { inputTextValue1, inputTextValue2, currentTask } = this.state;
 
 		return(
 			<Fragment>
 				<FormattedHeader
-					headerText="Newspack Components"
-					subHeaderText="Temporary demo of Newspack components"
+					headerText={ __( 'Newspack Components' ) }
+					subHeaderText={ __( 'Temporary demo of Newspack components' ) }
 				/>
+				<Checklist currentTask={ currentTask } progressBarText={ __( 'Your setup list' ) }>
+					<Task
+						title={ __( 'Set up membership' ) }
+						description={ __( 'Optimize your site for search engines and social media by taking advantage of our SEO tools. We\'ll walk you through important SEO strategies to get more exposure for your business.' ) }
+						buttonText={ __( 'Do it' ) }
+						completedTitle={ __( 'All set!' ) }
+						onClick={ () => this.setState( { currentTask: 1 } ) }
```

Comment:
- finally, modifying `onClick`, `onSkip`, `complete` and `current` **for all four of the `<Task/>` components**, like this:
```suggestion
						onClick={ () => this.completeTask( 0 ) }
						onSkip={ () => this.completeTask( 0 ) }
						complete={ _.includes( completedTasks, 0 ) }
						current={ 0 === currentTask }
```

The only problem now is that the use of the `<Task/>` component has all these repetitions of the keys, which actually looks worse now... Makes me think if the `<Task/>`s could maybe be created with the use of a map() function, maybe. This I'm not sure of.